test/Makefile

```diff
@@ -103,7 +103,7 @@ export SEP=$(subst /,\,/)
 DRUNTIME_PATH=..\..\druntime
 PHOBOS_PATH=..\..\phobos
 export DFLAGS=-I$(DRUNTIME_PATH)\import -I$(PHOBOS_PATH)
-export LIB=$(PHOBOS_PATH)
+#export LIB=$(PHOBOS_PATH)
```

Comment:
is this change intentional? I don't think we should change such a central place of the test makefile.